examples/advanced/landing-page/rivers.html

```diff
@@ -1,17 +1,19 @@
 <!DOCTYPE html>
 <html>
+
 <head>
   <title>US Population by County | CARTO</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta charset="UTF-8">
   <!-- Include CARTO VL JS -->
-  <script src="../../dist/carto-vl.js"></script>
+  <script src="../../../dist/carto-vl.js"></script>
```

Comment:
was this broken?

Reply:
@elenatorro is working in this new landing-page thing (a new folder has just been added)